python/ray/experimental/packaging/load_package.py

```diff
@@ -31,29 +31,32 @@ def load_package(config_path: str) -> "_RuntimePackage":
             example YAML in ``example_pkg/ray_pkg.yaml``.
 
     Examples:
-        >>> # Load from local.
-        >>> my_pkg = load_package("~/path/to/my_pkg.yaml")
 
-        >>> # Load from GitHub.
-        >>> my_pkg = ray.util.load_package(
-        ...   "https://raw.githubusercontent.com/user/repo/refspec"
-        ...   "/path/to/package/my_pkg.yaml")
+        .. code-block :: python
```

Comment:
Using `code-block:: python` so we can track this as an example that we should fix.